static/styles/zulip.scss

```diff
@@ -1085,11 +1085,7 @@ td.pointer {
 }
 
 .selected_message .messagebox-content {
-    box-shadow: inset 0px 0px 0px 2px hsl(215, 47%, 50%),
-                     -1px -1px 0px 0px hsl(215, 47%, 50%),
-                      1px 1px 0px 0px hsl(215, 47%, 50%),
-                     -1px 1px 0px 0px hsl(215, 47%, 50%),
-                      1px -1px 0px 0px hsl(215, 47%, 50%);
+    box-shadow: inset 0px 0px 0px 3px hsl(215, 47%, 50%);
```

Comment:
How did you verify that this commit doesn't make other potentially regression subtle visual changes?  Reasoning or testing are both useful for this sort of thing :)